arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -911,3 +912,35 @@ def rule_gfm_1_5_8(
                     label=label.textValue,
                     modelObject=label
                 )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_5_10(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.5.10] A non-numeric concept should not have a label with a numeric role
+    """
+    labelRelationshipSet = val.modelXbrl.relationshipSet(XbrlConst.conceptLabel)
+    if labelRelationshipSet is None:
+        return
+    for concept in val.modelXbrl.qnameConcepts.values():
+        if pluginData.isStandardTaxonomyUrl(concept.qname.namespaceURI, val.modelXbrl) or concept.isNumeric:
+            continue
```

Comment:
We determined that filers can create labels for standard taxonomy concepts, didn't we? Those should also be checked?